bench/Sender/Main.hs

```diff
@@ -56,7 +57,7 @@ main = do
     loadLogConfig logsPrefix logConfig
     setLocaleEncoding utf8
 
-    Right transport_ <- TCP.createTransport "0.0.0.0" "3432" TCP.defaultTCPParameters
+    Right transport_ <- TCP.createTransport "0.0.0.0" "127.0.0.1" "3432" TCP.defaultTCPParameters
```

Comment:
I think we ought to give only one address: the bind address. We can give `0.0.0.0`, and then patch nt-tcp so that, when a connection comes it, it uses `getSocketName` to determine the address at which the peer reached us (it won't be `0.0.0.0`).